integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/clients/rabbitmqclient/RabbitMQConsumerClient.java

```diff
@@ -39,8 +40,13 @@ public RabbitMQConsumerClient(String host) {
     }
 
     public void declareAndConnect(String exchangeName, String routeKey) throws IOException {
-        connection = factory.newConnection();
-        channel = connection.createChannel();
+        try {
+            connection = factory.newConnection();
+            channel = connection.createChannel();
+        } catch (TimeoutException e) {
+            e.printStackTrace();
```

Comment:
If we log the error only from here, It'll make an inconsistency in how we handled exceptions in test utils as all other places we have printing the stacktrace.